enable awq int4 in Gaudi (#1691)
huggingface/optimum-habana: examples/text-generation/README.md

--load_quantized_model_with_autogptq
```

### Running with UINT4 weight quantization using AutoAWQ

> Llama2-7b in UINT4 weight only quantization is enabled using [AutoAWQ](https://github.com/casper-hansen/AutoAWQ), which provides quantization capabilities in PyTorch.
> Currently, the support is for UINT4 inference of pre-quantized models only.

Reviewer: ```suggestion
Llama2-7b supports UINT4 weight-only quantization through [AutoAWQ](https://github.com/casper-hansen/AutoAWQ), which offers quantization capabilities in PyTorch. 
Currently, this support is limited to UINT4 inference of pre-quantized models only.
```

> ```bash

Reviewer 2: ```suggestion
Please run the following command to install AutoAWQ:
```bash
```

pip install autoawq
```

> You can run a *UINT4 weight quantized* model using AutoAWQ by adding the argument `--load_quantized_model_with_autoawq`.

Reviewer: ```suggestion
You can run a *UINT4 weight quantized* model using AutoAWQ by including the argument `--load_quantized_model_with_autoawq`.
```

> Here is an example to run a quantized model <quantized_awq_model>:

Reviewer: ```suggestion
Here is an example of how to run a quantized model <quantized_awq_model>:
```
